apps/gauzy/src/app/pages/settings/oauth-clients/oauth-client-form-dialog/oauth-client-form-dialog.component.html

```
@@ -0,0 +1,138 @@
+					<nb-option *ngFor="let t of clientTypes" [value]="t">{{ t }}</nb-option>
+				</nb-select>
+				<small class="hint" *ngIf="isEdit">
+					{{ 'OAUTH_CLIENTS.FORM.CLIENT_TYPE_IMMUTABLE' | translate }}
+				</small>
+			</div>
+
+			<div class="form-row" formArrayName="redirectUris">
+				<label class="label">{{ 'OAUTH_CLIENTS.FORM.REDIRECT_URIS' | translate }}</label>
+				<div class="array-row" *ngFor="let _ of redirectUris.controls; let i = index">
+					<input
+						nbInput
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)